drs/drs-assembly/subscribe-model.conf

```diff
@@ -1350,59 +1010,27 @@ Current configuration of filter wheel #5. Contains the following:
 }
 
 {
-  subsystem =  TCS
-  component =  tcs-tbd
-  name =  DPRES
-  requiredRate =  4
-  usage = "Dome pressure. FITS=DPRES"
-}
-
-{
-  subsystem =  TCS
-  component =  tcs-tbd
-  name =  DTEMP
-  requiredRate =  4
-  usage = "Dome temperature. FITS=DTEMP"
-}
-
-{
-  subsystem =  TCS
-  component =  tcs-tbd
-  name =  EXTDTEMP
+  subsystem =  ESEN
```

Comment:
Using ESEN seems like the best guess at present for getting all of these environmental data. Perhaps the individual subsystems will also provide their own aggregate values that would be better to use instead. I'd suggest talking to people in the controls group at TMT.